hazelcast/src/main/java/com/hazelcast/map/IMap.java

```diff
@@ -2455,6 +2455,47 @@ UUID addEntryListener(@Nonnull MapListener listener,
      */
     Collection<V> values(@Nonnull Predicate<K, V> predicate);
 
+    /**
+     * Returns an immutable collection locally owned values contained in this map.
+     * <p>
+     * <b>Warning:</b>
+     * <p>
+     * The collection is <b>NOT</b> backed by the map,
+     * so changes to the map are <b>NOT</b> reflected in the collection.
+     * <p>
+     * This method is always executed by a distributed query,
+     * so it may throw a {@link QueryResultSizeExceededException}
+     * if {@link ClusterProperty#QUERY_RESULT_SIZE_LIMIT} is configured.
+     *
+     * @return an immutable collection clone of the values contained in this map
+     * @throws QueryResultSizeExceededException if query result size limit is exceeded
+     * @see ClusterProperty#QUERY_RESULT_SIZE_LIMIT
```

Comment:
can you please add `@since 5.4.0` in the new methods' javadoc?

Reply:
I'll deal with this in a follow up PR.